### ISSUE TITLE
Sometimes unable to sap

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#5620]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Sometimes as rogue you are unable to sap target (target are not in combat)
**Expected behaviour**: Tell us what should happen instead
You always able to sap target that is not in combat
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
https://www.youtube.com/watch?v=9Js5t_zNnF8&t=0s&list=PLNMi27AamnGtQyiCfUO_zdPYv9krI5OPz&index=10
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/ 
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
